src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -8,7 +8,6 @@
             <index columns="mimeType"/>
             <index columns="size"/>
             <index columns="version"/>
-            <index columns="name"/>
```

Comment:
I think the name don't need an index because the database can't use an Index with '%value%' ? /cc @sulu/core-team

Reply:
Same as above, I am not compeletely sure if we never query for `value%`... And thinking of it, if you just remove that index it might make applications written by other developers a lot slower.